index.html

```diff
@@ -4,7 +4,7 @@
     <title>Fedora API Specification</title>
     <meta charset='utf-8'>
     <meta name="description" content="Fedora API Specification">
-    <script src='https://www.w3.org/Tools/respec/respec-w3c-common'
+    <script src='http://fedora.info/spec/respec/v14.0.13/respec-w3c-common'
```

Comment:
The local `file://` relative issue I described goes away if adding the `.js` suffix to `respec-w3c-common`.